src/devices/Llc200d3sh/Llc200d3sh.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="*.cs" />
+    <Compile Remove="samples\**" />
+    <None Include="README.md" />
+	  <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj" />
+	  <!--<ProjectReference Include="..\Common\Iot\Device\Common\CommonHelpers.csproj" />-->
```

Comment:
there shouldn't be any commented out lines (unless there is a comment explaining why it's there but most likely it should be removed anyway)